src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -108,4 +136,4 @@
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
I know GitHub warns about these, but VS removes these from project files, and I've stopped fighting it.